indico/htdocs/sass/partials/_boxes.scss

```diff
@@ -150,6 +150,7 @@
     padding: 0;
     &.titled > .i-box-content {
         margin-top: 0;
+        text-align: center;
```

Comment:
Are you sure that this isn't too broad (centering ALL i-box-content)? I believe you only want to center the "empty" message?